GiEnJul/Controllers/RecipientController.cs

```diff
@@ -0,0 +1,52 @@
+﻿using GiEnJul.Features;
+using Microsoft.AspNetCore.Mvc;
+using Serilog;
+using System;
+using System.Threading.Tasks;
+
+
+namespace GiEnJul.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class RecipientController : ControllerBase
+    {
+        private readonly IRecipientRepository _recipientRepository;
+        private readonly IPersonRepository _personRepository;
+        private readonly ILogger _log;
+
+        public RecipientController(IRecipientRepository recipientRepository, IPersonRepository personRepository, ILogger log)
+        {
+            _recipientRepository = recipientRepository;
+            _personRepository = personRepository;
+            _log = log;
+        }
+
+        [HttpPost]
+        public async Task<ActionResult<Entities.Recipient>> PostAsync([FromBody] Models.Recipient recipient)
+        {
+            _log.Debug("Adding recipient object: {@recipient}", recipient);
+            recipient.FamilyMembers.ForEach(person => person.PartitionKey = recipient.RowKey);
+            try
+            {
+                await _personRepository.InsertOrReplaceBatchAsync(recipient.FamilyMembers);
```

Comment:
This operation might not need its "own" try catch, as the default error-handler will have almost equal outcome.